### PR TITLE
feat: add hours warning to resource page

### PR DESCRIPTION
**Pull request by jacobvenable**
This PR does not close an issue.

## What does this PR do?

Adds a warning above the hours on each resource, instructing users to check website for more details.

screen short of warning, COVID-19 may affect normal business hours and operations. Please check their website for up-to-date info.

This is important because the situation surrounding COVID-19 is changing weekly. We aren't in a state where we can guarantee up-to-date information so it's best to warn them to contact the provider for more details.

## How does this PR make you feel? [:link:](http://giphy.com/categories/emotions/)

you've been wanred

### CODE REVIEW

File: packages/web/src/components/Resource.tsx [hunk 1]
```diff
@@ -162,17 +172,19 @@ export const Resource = () => {
...
-              <ScheduleIcon titleAccess={t("hoursOfOperation")} />
-            </ListItemIcon>
-            <ListItemText>
-              <Typography component="h2" variant="srOnly">
-                {t("hoursOfOperation")}
-              </Typography>
-              <Schedule schedule={resource.schedule} />
-            </ListItemText>
-          </ListItem>
+          <>
```
Comment (jollyjerr) [inline packages/web/src/components/Resource.tsx:175]: why this change?

**Review by jollyjerr — APPROVED** [review]
Screen Shot 2021-06-28 at 7 39 43 AM

LGTM 🚀 !

File: packages/web/src/components/Resource.tsx (hunk 1 above)
Comment (jacobvenable) [reply packages/web/src/components/Resource.tsx]: Ah derp, mistake on my part. [removed](https://github.com/CodeForBoulder/upswyng/pull/479/commits/778e8a84bf58a11eba79bf7fc33699d01f750b91)

File: packages/web/src/components/Resource.tsx [hunk 2]
```diff
@@ -110,6 +112,14 @@ export const Resource = () => {
         />
       )}
       <List component="div">
+        <ListItem component={Card}>
+          <ListItemIcon classes={listIconClasses}>
+            <WarningIcon color="secondary" />
+          </ListItemIcon>
+          <ListItemText>{`${t("covidWarning")} ${
+            !!resource.website ? t("checkWebsite") : ""
+          }`}</ListItemText>
```
Comment (0967622354toon) [inline packages/web/src/components/Resource.tsx:121]: Good
